README.md

```diff
@@ -24,6 +24,7 @@ The library is currenty able to convert the following pixel formats:
 | BGR                  | NV12                       |
 | NV12                 | BGRA                       |
 | RGB                  | BGRA                       |
+| I420                 | BGRA                       |
```

Comment:
At this point I think it's worth to sort this table lexicographically (key: source pixel format)